packages/compass-connect/src/stores/index.js

```diff
@@ -199,6 +201,7 @@ const Store = Reflux.createStore({
       this._setSyntaxErrorMessage(
         'Invalid schema, expected `mongodb` or `mongodb+srv`'
       );
+      log.info(mongoLogId(1001000025), 'Connection UI', 'Rejecting URL due to invalid schema');
```

Comment:
Is this very noisy? I'm wondering if it hits the log every time a key is pressed. Probably fine if it does that I suppose.

Reply:
Yeah, it’s a little noisy. :smile: We do debounce this, so it’s not terribly bad as long as the user isn’t actually typing each letter very carefully.

I think it’s fine, if it does become a problem we can probably even add a feature to the logging library that makes it implicitly remove adjacent duplicate entries?